cmd/earthly/org_cmds.go

```diff
@@ -122,6 +122,14 @@ func (app *earthlyApp) orgCmdsPreview() []*cli.Command {
 					UsageText:   "earthly org invite accept <invite-code>",
 					Action:      app.actionOrgInviteAccept,
 				},
+				{
+					Name:        "ls",
+					Aliases:     []string{"list"},
+					Usage:       "List all pending and accepted invitations",
```

Comment:
maybe this needs to be clarified that it's only sent invites:
```suggestion
					Usage:       "List all sent invitations (both pending and accepted)",
```